jobs/job_options.py

```diff
@@ -26,26 +26,31 @@
 
 
 class JobOptions(pipeline_options.GoogleCloudOptions):
-    """Option class for configuring the behavior of Oppia jobs."""
+    """Option class for configuring the behavior of Oppia jobs.
 
-    def __init__(
-            self,
-            project=feconf.OPPIA_PROJECT_ID,
-            region=feconf.GOOGLE_APP_ENGINE_REGION,
-            **kwargs):
+    Instances of this class should only change the arguments created in the
+    `_add_argparse_args` method:
```

Comment:
Can you make this more declarative? I.e. have an array of tuples that _add_argparse_args draws from programmatically, and change this to say that instances of this class should only change those args.